molecule/test13/verify.yml

```diff
@@ -13,8 +13,8 @@
     command: scontrol show config
     register: slurm_config
   - assert:
-      that: "item in slurm_config.stdout"
+      that: "item in slurm_config.stdout_lines | map('replace', ' ', '')"
```

Comment:
Might be worth adding brackets here?, I am unclear on the ordering here, I think its:
"item in (slurm_config.stdout_lines | map('replace', ' ', ''))"

Reply:
Done in 505661e - testing...